\

```diff
@@ -0,0 +1,61 @@
+Adding a log for proposals in the future
```

Comment:
what is this file : D ???

Reply:
Lol, I messed up my git workflow I guess. I'll remove it.